### PR TITLE
Milan convert episodes pages to ts

### PR DESCRIPTION
**Pull request by azurecorn**

### CODE REVIEW

File: model/episodeModel.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+}
+
+export interface EpisodeItem {
+  id: number;
+  name: string;
+  air_date: string;
+  episode: string;
+}
+
+export interface PageInfo {
```
Comment (bojanCrevar) [inline model/episodeModel.tsx:16]: use PaginationModel instead

File: model/formComponentInput.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { EpisodeItem } from "./episodeModel";
+
+interface FormComponentInput {
```
Comment (bojanCrevar) [inline model/formComponentInput.tsx:3]: Make this a single interface with optional initData parameter

**Review by bojanCrevar — APPROVED** [review]
Najs

File: model/episodeModel.tsx (hunk 1 above)
Comment (azurecorn) [reply model/episodeModel.tsx:16]: right, fixed

File: model/formComponentInput.tsx (hunk 2 above)
Comment (azurecorn) [reply model/formComponentInput.tsx:3]: fixed

**Pull request merged**